### DISCUSSION TITLE
How to run this project???

### DISCUSSION THREAD

**Discussion by aashi2912** [praveenscience/Internship-LMS-FrontEnd#47] [General]
not able to understand how to see this project properly so I can find my issue or add anything.

**Comment by ritik307** [comment]
@aashi2912 this project is in the planning phase. @praveenscience  is currently planning the project. Once everything is done then you can start contributing to it.

**Reply by aashi2912** [reply]
Okk Thank you.

**Reply by praveenscience** [reply]
@aashi2912 Let's get connected on [LinkedIn](https://www.linkedin.com/in/praveentech/) and discuss further too! 😁

**Reply by aashi2912** [reply]
sure. it's https://www.linkedin.com/in/aashithakkar29/

**Reply by praveenscience** [reply]
> sure. it's https://www.linkedin.com/in/aashithakkar29/

Done, please to accept! 😁
